script/Ciel/Day 8/08_01_CIEL08_1D.txt

```diff
@@ -527,7 +526,7 @@ What the hell is wrong with me?
 [sha:467325d03d3f0b2e8a2a66ca5592938c66168e11]{
 -- Page 129, Offset 18600.
 -- 俺は虫以下の人でなしだった。
-I'm scum. I'm lower than an insect.
+I'm a mistake――less than even an insect.
```

Comment:
Hmm kinda felt this the first time though too but I think 'lower' might work better than 'less' here. Or some third alternative. Maybe just Gregor Samsa it?
> I'm a mistake―a [disgusting,despicable] cockroach.

Reply:
I think this line is changed in the 'petri notes' one anyway, so I guess I'll revert it so it doesn't cause conflict.